app/views/waste_exemptions_engine/main_people_forms/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_for(@main_people_form) do |f| %>
```

Comment:
It's being picky, but I think this refactoring of the `app/views/waste_exemptions_engine/main_people_forms/new.html.erb` to use a partial is worthy of a PR by itself.

Then this one truly just becomes about whitespace and error class changes.